cirq-rigetti/requirements.txt

```diff
@@ -0,0 +1,18 @@
+attrs==20.3.0
+certifi==2021.5.30
+h11==0.9.0
+httpcore==0.11.1
+httpx==0.15.5
+idna==2.10
+iso8601==0.1.14
+pydantic==1.8.2
+pyjwt==1.7.1
+python-dateutil==2.8.1
+qcs-api-client==0.8.0
+retrying==1.3.3
+rfc3339==6.2
+rfc3986==1.5.0
+six==1.16.0
+sniffio==1.2.0
+toml==0.10.2
+typing-extensions==3.10.0.0
```

Comment:
The way Cirq uses requirements.txt is a bit unorthodox - we should remove as much of the constraints as possible as we are using requirements.txt as the `install_requires` instruction in setup.py. This does mean that our continuous integration keeps changing as new versions are released of our dependencies, but it didn't cause too many problems just yet. See #3441 for detailed discussions.

Reply:
Ok, I've dropped some dependencies (I see pyQuil itself was already a dependency for the Quil conversion) and loosened constraints to `~=`. Let me know if that suffices of if there is something additional that'd facilitate management.